solr/core/src/test/org/apache/solr/cluster/placement/plugins/AffinityPlacementFactoryTest.java

```diff
@@ -1443,60 +1445,6 @@ private void testSpreadDomains(boolean hasExistingCollection) throws Exception {
     }
   }
 
-  @Test
-  @SuppressWarnings("SelfComparison")
-  public void testCompareSpreadDomainWithNodes() {
```

Comment:
[Q] Are we deleting this test because the functionality will go away in 10.x?  Or has it moved somewhere that I've missed...

Reply:
I deleted it, because it uses a low level class that no longer exists, and is very difficult to be replicated using the new logic/classes.